package.json

```diff
@@ -78,7 +78,7 @@
     "react-native-linear-gradient": "2.8.3",
     "react-native-localize": "3.1.0",
     "react-native-math-view": "3.9.5",
-    "react-native-navigation": "7.39.1",
+    "react-native-navigation": "7.40.0",
```

Comment:
Is this version the same as in main?

Reply:
Yes, it is. https://github.com/mattermost/mattermost-mobile/blob/c6725ab6d3506f0a5216d8dec85fcf83b6f5bb6f/package.json#L83